pyvista/utilities/helpers.py

```diff
@@ -117,6 +117,11 @@ def convert_string_array(arr, name=None):
 
     """
     if isinstance(arr, np.ndarray):
+        # VTK default fonts only support ASCII. See https://gitlab.kitware.com/vtk/vtk/-/issues/16904
+        if not str(arr).isascii():  # avoids segfault
```

Comment:
Slight nitpick here and also in the tests:
```suggestion
        if not ''.join(arr).isascii():  # avoids segfault
```

Because otherwise we're checking a stringified array:
```py
>>> arr = np.array(list('árvíztűrő tükörfúrógép'))

>>> arr
array(['á', 'r', 'v', 'í', 'z', 't', 'ű', 'r', 'ő', ' ', 't', 'ü', 'k',
       'ö', 'r', 'f', 'ú', 'r', 'ó', 'g', 'é', 'p'], dtype='<U1')

>>> str(arr)
"['á' 'r' 'v' 'í' 'z' 't' 'ű' 'r' 'ő' ' ' 't' 'ü' 'k' 'ö' 'r' 'f' 'ú' 'r'\n 'ó' 'g' 'é' 'p']"

>>> ''.join(arr)
'árvíztűrő tükörfúrógép'
```

What we have now is also correct because the fluff in the string representation is ASCII, but it's slightly cleaner to only check the contents.